lib/tasks/data/sanitize_data.rake

```diff
@@ -0,0 +1,56 @@
+require 'highline'
+
+namespace :ofn do
+  namespace :data do
+    desc 'Sanitize data'
+    task sanitize: :environment do
+      guard_and_warn
+
+      Spree::User.update_all("email = concat(id, '_ofn_user@example.com'),
+                              login = concat(id, '_ofn_user@example.com'),
+                              unconfirmed_email = concat(id, '_ofn_user@example.com')")
+      Spree::Customer.update_all("email = concat(id, '_ofn_customer@example.com'),
```

Comment:
This line means that customer emails won't be in sync with user emails any more. They should be the same. Maybe we could do something like this:
```rb
Spree::Customer.where("user_id IS NULL").update_all("email = concat(id, '_ofn_customer@example.com')...")
Spree::Customer.where("user_id IS NOT NULL").update_all("email = concat(user_id, '_ofn_user@example.com')...")
```

Reply:
But in this case we lose all the examples where customer data is not in sync... It's a bit more complicated if we wanted to keep the entropy. So maybe we shouldn't bother unless there is a problem with this.

Reply:
I like your suggestion to keep them in sync, I have added it in a new commit. I have tested it locally to validate it won't make the task take 3 days to run :-)

Reply:
also, I dont understand how Spree::Customer got into here, it's just Customer. I could swear this was tested... well, now it's tested for sure!